File: src/main/kotlin/com/snsystems/kotlinLang/Features.java
package com.snsystems.kotlinLang;

public class Features {
    public static void main(String[] args) {
        String greeting = "Hello";
        System.out.println(greeting + " World");

        DealJavaDTO dealJavaDTO = new DealJavaDTO(1, "PRIME", "Premium Deal", "Active");
        System.out.println("Deal Code : " + dealJavaDTO.getDealCode());
        System.out.println("Deal Name : " + dealJavaDTO.getDealName());
        System.out.println("Status: " + dealJavaDTO.getStatus());
		
		
		if (greeting != null) {
			System.out.println(greeting);
		}
    }
}

File: src/main/kotlin/com/snsystems/kotlinLang/Features.kt
package com.snsystems.kotlinLang

import com.snsystems.kotlinLang.DealKotlinDTO

fun main() {
    val greeting = "Hello"
    println("$greeting World")
    println(greeting.plus(" World"))

    var dealKotlinDTO = DealKotlinDTO(1, "PRIME", "Premium Deal", "Active")
    println("Deal Code :  $dealKotlinDTO")
    println("Deal Name : $dealKotlinDTO.getDealName()")
    println("Status: $dealKotlinDTO.getStatus()")

	easyNotNull(null)
    easyNotNull("Sathya")
}

fun easyNotNull(s: String?) {
s?.let {
	println(s)
}